libclc/CMakeLists.txt

```diff
@@ -358,3 +399,7 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
     endif()
   endforeach( d )
 endforeach( t )
+
+if( NOT LIBCLC_STANDALONE_BUILD )
+  add_subdirectory( test )
```

Comment:
There is no `CMakeLists.txt` in `<llvm-project>/llvm/libclc/test`.

Reply:
Ach yeah sorry, I have another path locally which adds some LIT tests - this should be in that one. I'll remove it for now, and we can maybe decide whether or not to add LIT testing to this PR or not.